action.yaml

```diff
@@ -4,191 +4,506 @@ branding:
   icon: upload-cloud
   color: red
 inputs:
-  # Checkout
-  checkout:
-    description: 'Specifies if this action should checkout the code'
-    required: false
-    default: 'true'
-  
-  # AWS
+  # AWS Specific
   aws_access_key_id:
     description: 'AWS access key ID'
-    required: true
+    required: false
   aws_secret_access_key:
     description: 'AWS secret access key'
-    required: true
+    required: false
   aws_session_token:
     description: 'AWS session token'
     required: false
   aws_default_region:
     description: 'AWS default region'
     default: us-east-1
     required: false
-  aws_ami_id:
-    description: 'AWS AMI ID. Will default to the latest Ubuntu 22.04 server image (HVM) '
+  aws_resource_identifier:
+    description: 'Set to override the AWS resource identifier for the deployment.  Defaults to `${org}-{repo}-{branch}`.  Use with destroy to destroy specific resources.'
+    required: false
+  aws_additional_tags: # additional_tags
+    description: 'A JSON object of additional tags that will be included on created resources. Example: `{"key1": "value1", "key2": "value2"}`'
+    required: false
+
+  # GitHub Commons main inputs
+  checkout:
+    description: 'Specifies if this action should checkout the code'
+    required: false
+    default: 'true'
+  bitops_code_only:
+    description: 'Will run only the generation phase of BitOps, where the Terraform and Ansible code is built.'
+    required: false
+  bitops_code_store:
+    description: 'Store BitOps code as a GitHub artifact'
+    required: false
+  tf_stack_destroy: # stack_destroy
+    description: 'Set to "true" to Destroy the stack through Terraform.'
+    required: false
+  tf_state_file_name:
+    description: 'Change this to be anything you want to. Carefull to be consistent here. A missing file could trigger recreation, or stepping over destruction of non-defined objects.'
+    required: false
+  tf_state_file_name_append:
+    description: 'Append a string to the tf-state-file. Setting this to `unique` will generate `tf-state-aws-unique`. Can co-exist with the tf_state_file_name variable. '
     required: false
   tf_state_bucket:
     description: 'AWS S3 bucket to use for Terraform state. Defaults to `${org}-${repo}-{branch}-tf-state`'
     required: false
   tf_state_bucket_destroy:
-    description: 'Force purge and deletion of S3 bucket defined. Any file contained there will be destroyed. `stack_destroy` must also be `true`'
+    description: 'Force purge and deletion of S3 bucket defined. Any file contained there will be destroyed. `tf_stack_destroy` must also be `true`'
```

Comment:
"Only evaluated when `tf_stack_destroy` is also true, so it is safe to leave this enabled when standing up your stack."